Src/CSharpier/ConfigurationOptions.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace CSharpier
+{
+    public class ConfigurationOptions
+    {
+        public HashSet<string> Exclude { get; set; }
+
+        public ConfigurationOptions(string[]? exclude)
+        {
+            if (exclude != null)
+            {
+                this.Exclude = exclude.Select(o => o.Replace("\\", "/"))
+                    .ToHashSet();
+            }
+            else
+            {
+                this.Exclude = new HashSet<string>();
+            }
+        }
+
+        public static ConfigurationOptions Create(string rootPath)
+        {
+            var directoryInfo = new DirectoryInfo(rootPath);
+
+            var nextPath = Path.Combine(directoryInfo.FullName, ".csharpierrc");
+            if (File.Exists(nextPath))
+            {
+                return JsonConvert.DeserializeObject<ConfigurationOptions>(
```

Comment:
Relatively minor point, but I would much prefer yaml over json - it doesn't require a quotes/commas and _allows comments_ in the spec, so all editors support them.

Reply:
I hate that json doesn't allow comments but don't use yaml much so usually default to json. I think we could pretty easily support both and updated https://github.com/belav/csharpier/issues/43

Reply:
You should definitely use YAML more. :grin: 
I think you'll start to like it - for hand-written files, it's much nicer than JSON.